proto/c1/connectorapi/baton/v1/baton.proto

```diff
@@ -49,6 +49,7 @@ message Task {
     repeated google.protobuf.Any annotations = 1;
     bool skip_expand_grants = 2;
     bool skip_entitlements_and_grants = 3;
+    bool skip_grants = 4;
```

Comment:
Service mode doesn't support partial sync, only full and resource_only. So if skip_grants requires a partial sync, this field must always be false.

I think for now it should just not exist on SyncFullTask. When we add support for partial sync in service mode, then it can be added.